src/KeyVault/KeyVault/help/Export-AzKeyVaultSecurityDomain.md

```diff
@@ -33,7 +33,7 @@ Exports the security domain data of a managed HSM for importing on another HSM.
 
 ### Example 1
 ```powershell
-Export-AzKeyVaultSecurityDomain -Name testmhsm -Certificates {pathOfCertificates}/sd1.cer, {pathOfCertificates}/sd2.cer, {pathOfCertificates}/sd3.cer -OutputPath {pathOfOutput}/sd.ps.json -Quorum 2
+Export-AzKeyVaultSecurityDomain -Name testmhsm -Certificates {{pathOfCertificates}/sd1.cer, {pathOfCertificates}/sd2.cer, {pathOfCertificates}/sd3.cer} -OutputPath {{pathOfOutput}/sd.ps.json} -Quorum 2
```

Comment:
In the original example, `{}` is not powershell gramma. It means that `pathOfCertificates` is a placeholder that user needs to replace with the real path.
If `{}` made our tooling think the example is wrong, I suggest replace `{pathOfCertificates}/sd1.cer` to `./sd1`, and `{pathOfOutput}/sd.ps.json` to `sd.ps.json`.

Reply:
Thanks. We will do that.